src/containers/FooterContainer/FooterContainer.vue

```diff
@@ -220,7 +220,8 @@ export default {
             },
             {
               text: 'KeepKey',
-              href: 'http://keepkey.go2cloud.org/aff_c?offer_id=1&aff_id=5561'
+              href:
+                'http://keepkey.myshopify.com?afmc=pi&utm_campaign=pi&utm_source=leaddyno&utm_medium=affiliate'
```

Comment:
we also have to edit this page,
https://www.myetherwallet.com/hardware-wallet-affiliates

can we make a js file where we can reference all these urls. That way we just have to import it and only change in one place